src/styles/base.scss

```diff
@@ -68,6 +68,13 @@ a {
   font-size: 28px;
   line-height: 110%;
   text-underline-offset: 8px;
+
+
+  &:hover {
+    text-underline-offset: 7px;
+    text-shadow: 0px -4px 7px #979797;
```

Comment:
I like the `underline` + `transition` for our `:hover` effect until we can do a fancier iteration, but we should remove the `text-shadow` property.